app/controllers/admin/schools_controller.rb

```diff
@@ -27,5 +28,19 @@ def show
     def load_school
       @school = School.eligible.friendly.find(params[:id])
     end
+
+    def search_query(relation)
+      if @query.present?
+        search_value = "%#{@query}%"
+
+        relation
+          .left_joins(:induction_coordinators)
+          .where(School.arel_table[:name].matches(search_value)
```

Comment:
What does `arel_table` do exactly?

Reply:
@kicferk1 ActiveRecord querying language is build on top of arel: https://github.com/rails/arel. `arel_table` returns instance of `Arel::Table`, `School.arel_table[:name]` returns `Arel::Attributes::Attributes` and `.matches` returns `Arel::Nodes::Matches` (which is a subclass of `Arel::Nodes::NodeExpression`) and `mathches(str).to_sql` returns nice `"schools"."name" ILIKE 'hello'`. Arel is much more generic than hash and allows constructing all sort of crazy sql than standard AR doesn't. 

However, I am not sure if it work here. I#d also prefer using ransack for searching.

Reply:
Yes, what @BroiSatse said ☝️ and yes it does work here.

Reply:
cool, good to know :)